### ISSUE TITLE
Allow for people to use Social Media Logins

### ISSUE THREAD

**Issue by matthewtrask** [matthewtrask/elephpant-traders#1]
Right now, we can only let users create a username and password, but we should look into Laravel Socialite and let them use Twittier/Facebook and maybe Github.

**Comment by VictorAlagwu** [comment]
Can i work on this

**Comment by matthewtrask** [comment]
absolutely! I was planning on using Laravel Socialite to do most of the work.

**Comment by matthewtrask** [comment]
I think we should look at the following: 

* Facebook
* Twitter 
* Github

**Comment by VictorAlagwu** [comment]
ok, will do that with socialite{That is if someone is not working on it}

**Comment by matthewtrask** [comment]
So far no one else is working on it. Im the only one so far working on the project.

**Comment by VictorAlagwu** [comment]
ok, will try and work on it
